pkg/metrics/metrics.jsx

```diff
@@ -387,16 +390,33 @@ class CurrentMetrics extends React.Component {
                     const is_user = k.match(/^user.*user@\d+\.service.+/);
                     const label = k.replace(/.*\//, '').replace(/\.service$/, '');
                     // only keep cgroup basenames, and drop redundant .service suffix
-                    merged.push([label, v, is_user]);
+                    merged.push([label, v, is_user, false]);
+                }
+                // filter out podman containers
+                const matches = k.match(podmanSystemRe);
+                if (matches && v) {
+                    // truncate to 12 chars like the podman output
+                    const containerid = matches.groups.containerid.substr(0, 12);
+                    const is_user = k.match(/^user.*user@\d+\.service.+/);
+                    merged.push([containerid, v, is_user, true]);
                 }
             });
             merged.sort((a, b) => b[1] - a[1]);
             return merged.slice(0, n);
         }
 
-        function serviceRow(name, value, is_user) {
+        function serviceClickHandler(name, is_user, is_container) {
+            if (is_container && cockpit.manifests && cockpit.manifests.podman) {
+                cockpit.jump("/podman");
```

Comment:
This needs to handle the "container && !manifests.podman" case separately -- it can't be a link to the services page then.

Reply:
Yes, I think we don't want it to be a link then as that looks confusing, maybe we even want to show a tooltip that users should install cockpit-podman?

Reply:
Hmm turns out that is not too simple as there is some issue with the just adding a <Tooltip> around the button. It seems that react is being "smart" by drawing the <tooltip> once and then just changing the text which means tooltips initially show up but after the data changes it does not.